packages/sdk/base/src/currencies.ts

```diff
@@ -1,10 +1,3 @@
-/** @deprecated use StableToken and Token */
```

Comment:
Seems appropriate to remove code that has been marked a `@deprecated` 2 years ago (April 2022) in https://github.com/celo-org/developer-tooling/commit/5cfd16214ca7ef7a7ff428c7d397933b3e1eeb51 👍